.travis/publishLicenseList.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Prefer bash specific since that is what I am testing against

Reply:
> Prefer bash specific since that is what I am testing against

You can get cheap POSIX-ish tests using `bash --posix …`.  But really, the guard here is to develop based on the POSIX spec instead of relying on your current implementation.  If you're not aware of any Bash-isms, I'd prefer we use `/bin/sh`, I'll review these for conformance, and we'll field bug reports for other shells if/when they come in.

Reply:
If you don't mind reviewing and/or testing, go ahead and review and make a PR when completed.